graphql/pupil/mutations.ts

```diff
@@ -72,6 +73,15 @@ export class PupilUpdateInput {
     matchReason?: string;
 }
 
+@InputType()
+export class PupilScreeningUpdateInput {
+    @Field(() => PupilScreeningStatus)
```

Comment:
Do we always want to set both status AND comment? Maybe we should make both optional here (`, { nullable: true }` in the field) so that one can also just set the comment or the status